File: PROVAS/ProvaAtual/ProvaPraticaTerca-3/Pokemon.java
import java.util.*;

public class Pokemon {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        
        Set<String> capturados = new HashSet<>();

        for (int i = 0; i < N; i++) {
            capturados.add(sc.next());
        }

        int faltam = 151 - capturados.size();

        System.out.println("Falta(m) " + faltam + " pokemon(s)");
    }
}

File: PROVAS/ProvaAtual/ProvaPraticaTerca-3/Pokemon.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main() {
    int N;
    scanf("%d", &N);

    char pokemons[1000][1001];
    int count = 0;

    for (int i = 0; i < N; i++) {
        char nome[1001];
        scanf("%s", nome);

        int existe = 0;
        for (int j = 0; j < count; j++) {
            if (strcmp(nome, pokemons[j]) == 0) {
                existe = 1;
                break;
            }
        }

        if (!existe) {
            strcpy(pokemons[count], nome);
            count++;
        }
    }

    int faltam = 151 - count;

    printf("Falta(m) %d pokemon(s)\n", faltam);

    return 0;
}
